launcher/minecraft/mod/ModFolderModel.cpp

```diff
@@ -97,8 +101,24 @@ QVariant ModFolderModel::data(const QModelIndex &index, int role) const
         }
 
     case Qt::ToolTipRole:
+        if (column == NAME_COLUMN) {
+            if (at(row)->isSymLinkUnder(instDirPath())) {
+                return m_resources[row]->internal_id() +
+                    tr("\nWarning: This resource is symbolicly linked from elsewhere. Editing it will also change the origonal") +
```

Comment:
Look above

Reply:
```suggestion
                    tr("\nWarning: This resource is symbolically inked from elsewhere. Editing it will also change the original.") +
```